javascript/javascript3/week1/cityWeather.js

```diff
@@ -0,0 +1,155 @@
+
+                // Display the data
+                // This data should be showed in your app:
+
+                // The chosen city
+                // Temperature
+                // Icon for the weather type
+                // Wind speed
+                // How clowdy it is
+                // When sunrise and sunset is
+                // Optional a map showing where the city is located
+
+function weatherToMyChoice(){
```

Comment:
Try to give better name for function, It should be action or verb describing what it does.